apertium-kmr-eng.kmr-eng.dix

```diff
@@ -15199,6 +15201,8 @@
       <e><p><l>êdî<s n="adv"/></l><r>from<b/>now<b/>on<s n="adv"/></r></p></e>
       <e><p><l>êdî<s n="adv"/></l><r>any<b/>more<s n="adv"/></r></p></e>
       <e><p><l>êdî<s n="adv"/></l><r>no<b/>more<s n="adv"/></r></p></e>
+      <e><p><l>tuxwa<s n="adv"/></l><r>for<b/>God's<b/>sake<s n="adv"/></r></p></e>
```

Comment:
Where did you get these translations?

Reply:
I originally got these from an online forum; however, upon reconsideration, I've removed them.